dspace/config/modules/rest.cfg

```diff
@@ -5,4 +5,137 @@
 #---------------------------------------------------------------#
 
 # record stats in DSpace statistics module
-stats = true
\ No newline at end of file
+stats = true
+
+#------------------------------------------------------------------#
+# REST API Reporting Tools                                         #
+#------------------------------------------------------------------#
+# This project is intended as an optional add-on to DSpace to provide 
+# Quality Control Reporting for Collection Managers.       
+#
+# See https://github.com/DSpace-Labs/DSpace-REST-Reports
+#                                                                  
+# These reports utilize the DSpace REST API to provide a Collection 
+# Manager with
+#  - an overview of their collections
+#  - a tool to query metadata for consistency
+#
+# When deploying the DSpace REST API, and institution may choose to 
+# make the API publicly accessible or to restrict access to the API. 
+# If these reports are deployed in a protected manner, the reporting 
+# tools can be configured to bypass DSpace authorization when 
+# reporting on collections and items. 
+
+##### Enable/disable authorization for the reporting tools. #####
+# By default, the DSpace REST API will only return communities/collections/items that are accessible to a particular user.
+# If the REST API has been deployed in a protected manner, the reporting tools can be configured to bypass authorization checks.
+# This will allow all items/collections/communities to be returned to the report user.
+# Set the rest-reporting-authenticate option to false to bypass authorization
+rest-reporting-authenticate = false
```

Comment:
@tdonohue , I agree.  I will update the PR to disable this setting.  If we are able to get the other auth methods working with rest, it will make sense to remove this option entirely.